showPartsOfTest.js

```diff
@@ -15,6 +21,19 @@ $("#thanks_for_participating").hide();
 $("#error_uploading").hide(); 
 
 
+function handleEventShowConsent(e) {
+  $("#html_embed_widget_11018").hide(hideShowTime);  //Unity window
```

Comment:
Do we need to hide the Unity window every time? Shouldn't it be enough to hide it at start?